roles/dotfiles/templates/zshrc.j2

```diff
@@ -14,13 +14,20 @@ DEFAULT_USER="{{ lookup('env','USER') }}"
 
 export PATH=$HOME/bin:/usr/local/bin:/usr/local/sbin:$PATH
 
+# OpenSSL
+export PATH="{{ lookup('pipe', 'brew --prefix openssl') }}/bin:$PATH"
```

Comment:
Avoid issues when debugging openssl issues where without this line :

```
LibreSSL 2.8.3
built on: date not available
platform: information not available
options:  bn(64,64) rc4(16x,int) des(idx,cisc,16,int) blowfish(idx)
compiler: information not available
OPENSSLDIR: "/private/etc/ssl"
```

With line : 

```
OpenSSL 1.1.1g  21 Apr 2020
built on: Tue Apr 21 13:30:00 2020 UTC
platform: darwin64-x86_64-cc
options:  bn(64,64) rc4(16x,int) des(int) idea(int) blowfish(ptr)
compiler: clang -fPIC -arch x86_64 -O3 -Wall -DL_ENDIAN -DOPENSSL_PIC -DOPENSSL_CPUID_OBJ -DOPENSSL_IA32_SSE2 -DOPENSSL_BN_ASM_MONT -DOPENSSL_BN_ASM_MONT5 -DOPENSSL_BN_ASM_GF2m -DSHA1_ASM -DSHA256_ASM -DSHA512_ASM -DKECCAK1600_ASM -DRC4_ASM -DMD5_ASM -DAESNI_ASM -DVPAES_ASM -DGHASH_ASM -DECP_NISTZ256_ASM -DX25519_ASM -DPOLY1305_ASM -D_REENTRANT -DNDEBUG
OPENSSLDIR: "/usr/local/etc/openssl@1.1"
ENGINESDIR: "/usr/local/Cellar/openssl@1.1/1.1.1g/lib/engines-1.1"
Seeding source: os-specific
```

Here the `OPENSSLDIR` is correct.